scripts/stage-release.sh

```diff
@@ -160,13 +158,21 @@ git commit --message "Release version ${RELEASE_TAG}" --signoff
 
 git tag -f "${RELEASE_TAG}"
 
-git push "${REPOSITORY}" "${RELEASE_TAG}" ${GIT_DRYRUN:-}
+git push "${REPOSITORY}" "${RELEASE_TAG}"
```

Comment:
Should we also tag the commit with the `run-id-` for traceability?

Reply:
I'm actually wondering if the release phase should not tag the release with the `${RELEASE_TAG}`.  It should tag with the run-id.  It should be a responsibility of the release phase to create the real tag. That would resolve #1086.  I don't want to make this change part of this PR.